lib/ansible/modules/cloud/azure/azure_rm_appserviceplan.py

```diff
@@ -38,10 +38,10 @@
 
     sku:
         description:
-            - The pricing tiers, e.g., F1, D1, B1, B2, B3, S1, P1, P1V2 etc.
+            - The pricing tiers, e.g., C(F1), C(D1), C(B1), C(B2), C(B3), C(S1), C(P1), C(P1V2) etc.
             - Please see U(https://azure.microsoft.com/en-us/pricing/details/app-service/plans/) for more detail.
             - For linux app service plan, please see U(https://azure.microsoft.com/en-us/pricing/details/app-service/linux/) for more detail.
-
+            - Sku needs to be configured at creation time.
```

Comment:
not necessary, user could update sku to scale up or down.

Reply:
Thanks for your review, already remove this line!